RPIPs/RPIP-46.md

```diff
@@ -28,42 +28,47 @@ A fixed percentage of RPL inflation is currently being used to fund ongoing main
-   5. `increase_no_share_seal_count`: 6
-   6. `allowlisted_controllers`: []
-10. Prior to the release of Saturn 1, a ranked-choice vote MUST be held to select a mechanism for surplus revenue share
-    1. The choices MAY include [RPIP-45: RPL Burn](RPIP-45.md), [RPIP-50: RPL LP](RPIP-50.md), or using higher `voter_share`
-    2. If higher `voter_share` is selected, the entirety of the ["Specification taking effect with Saturn 2"](#specification-taking-effect-with-saturn-2) section below SHALL be deleted
-    3. The selected mechanism MAY be implemented in Saturn 1
-       1. If not, the revenue SHALL be held in reserve until the mechanism is implemented. In such a case, once the mechanism is implemented, the revenue SHALL be distributed in the same amount of time it took to build up, or faster.
-    4. The selected mechanism MUST be implemented in Saturn 2
-    5. When implemented, `surplus_share` SHOULD be renamed to something more specific
+6. The `node_operator_commission_share_council_adder` setting SHALL only be set to values within 0% and `max_node_operator_commission_share_council_adder` (inclusive)
+7. The `node_operator_commission_share_council_adder` setting SHALL be controllable by the security council without requiring a delay
+8. The security council SHOULD increment `node_operator_commission_share_council_adder` by 0.5% if the deposit pool is over half-full for the majority of a 2-week period with a constant `node_operator_commission_share + node_operator_commission_share_council_adder`
+9. `voter_share` SHALL NOT be controllable by pDAO vote and SHALL be initialized to 5%
+10. The initial pDAO settings SHALL be:
+    1. `node_operator_commission_share`: 3.5%
+    2. `node_operator_commission_share_council_adder`: 0%
+    3. `surplus_share`: 5.5%
+    4. `max_node_operator_commission_share_council_adder`: 1%
+    5. `allowlisted_controllers`: []
+
+### Inflation
+1. Inflation settings SHALL be modified to retain inflation to the DAOs and eliminate inflation to NOs
+   1. `rpl.inflation.interval.rate` SHALL be set to `1000040763630249500` (1.5% per year)
+   2. Node Operators (`rocketClaimNode`) allocation SHALL be set to 0%
+   3. pDAO (`rocketClaimDAO`) allocation SHALL be set to 95%
+   4. oDAO (`rocketClaimTrustedNode`) allocation SHALL be set to 5%
+
+### Surplus revenue share vote
+Prior to the release of Saturn 1, a ranked-choice vote MUST be held to select a mechanism for surplus revenue share
+1. The choices MAY include [RPIP-45: RPL Burn](RPIP-45.md), [RPIP-50: RPL LP](RPIP-50.md), using higher `voter_share`, and/or additional options
+2. If higher `voter_share` is selected:
+   1. The entirety of the ["Specification taking effect with Saturn 2"](#specification-taking-effect-with-saturn-2) section below SHALL be deleted
+   2. `voter_share` SHALL become controllable by pDAO vote; however, it SHALL require a supermajority vote with at least 75% of the vote in support of any increase 
```

Comment:
what is the point of having 75% when `voter_share` won? The conflict is between `surplus_share` and `voter_share`, not `no_share`. 
If we need 75% rule here, why don't we have it for changing `no_share` vs `surplus_share` from the start or for other selections of `surplus_share` ?

Reply:
Yep - deleting that line